app/controllers/search_controller.rb

```diff
@@ -169,6 +169,8 @@ def lookup_search_context
         context_obj = Category.find_by(id: search_context[:id].to_i)
       elsif 'topic' == search_context[:type]
         context_obj = Topic.find_by(id: search_context[:id].to_i)
+      elsif 'tag' == search_context[:type]
+        context_obj = Tag.where_name(search_context[:id]).first
```

Comment:
Minor but API wise this is a bit confusing. Usually when we hand over "ids" you would expect this to be the Tag id. 

Perhaps we populate `search_context[:name]` in this case?

Reply:
Hmm yeah good point